examples/Remarketing/UploadOfflineConversions.php

```diff
@@ -0,0 +1,157 @@
+                $options[ArgumentNames::CONVERSION_ACTION_ID] ?: self::CONVERSION_ACTION_ID,
+                $options[ArgumentNames::GCL_ID] ?: self::GCL_ID,
+                $options[ArgumentNames::CONVERSION_TIME] ?: self::CONVERSION_TIME,
+                $options[ArgumentNames::CONVERSION_VALUE] ?: self::CONVERSION_VALUE
+            );
+        } catch (GoogleAdsException $googleAdsException) {
+            printf(
+                "Request with ID '%s' has failed.%sGoogle Ads failure details:%s",
+                $googleAdsException->getRequestId(),
+                PHP_EOL,
+                PHP_EOL
+            );
+            foreach ($googleAdsException->getGoogleAdsFailure()->getErrors() as $error) {
+                /** @var GoogleAdsError $error */
+                printf(
+                    "\t%s: %s%s",
+                    $error->getErrorCode()->getErrorCode(),
+                    $error->getMessage(),
+                    PHP_EOL
+                );
+            }
+        } catch (ApiException $apiException) {
+            printf(
+                "ApiException was thrown with message '%s'.%s",
+                $apiException->getMessage(),
+                PHP_EOL
+            );
+        }
+    }
+
+    /**
+     * Runs the example.
+     *
+     * @param GoogleAdsClient $googleAdsClient the Google Ads API client
+     * @param int $customerId the customer ID
+     * @param int $conversionActionId the ID of the conversion action to upload to
+     * @param string $gclId the GCLID for the conversion (should be newer than 30 days)
+     * @param string $conversionTime the date and time of the conversion (should be after the
+     * click time)
+     * @param float $conversionValue the value of the conversion
+     */
+    public static function runExample(
+        GoogleAdsClient $googleAdsClient,
+        int $customerId,
+        int $conversionActionId,
+        string $gclId,
+        string $conversionTime,
+        float $conversionValue
+    ) {
+        // Creates a click conversion.
+        $clickConversion = new ClickConversion([
+            'conversion_action' => new StringValue([
+                'value' => ResourceNames::forConversionAction($customerId, $conversionActionId)
+            ]),
+            'gclid' => new StringValue(['value' => $gclId]),
+            'conversion_value' => new DoubleValue(['value' => $conversionValue]),
+            'conversion_date_time' => new StringValue(['value' => $conversionTime]),
+            'currency_code' => new StringValue(['value' => 'USD']),
+        ]);
+
+        // Issues a request to upload the click conversion.
+        $conversionUploadServiceClient = $googleAdsClient->getConversionUploadServiceClient();
+        /** @var UploadClickConversionsResponse $response */
+        $response = $conversionUploadServiceClient->uploadClickConversions(
+            $customerId, [$clickConversion], ['partialFailure' => true]
```

Comment:
Is this PSR-2 compliant?

Reply:
Good catch, only one argument per line is permitted so I changed